### PR TITLE
Backport of the "Update with the latest chain-libs", "graphql: add total_count to Block::transactions"

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
Backporting the following PRs https://github.com/input-output-hk/jormungandr/pull/3684, https://github.com/input-output-hk/jormungandr/pull/3685, https://github.com/input-output-hk/jormungandr/pull/3683 and updated `chain-libs` to the https://github.com/input-output-hk/chain-libs/pull/687

### CODE REVIEW

**Comment by Zeegomo** [comment]
Is https://github.com/input-output-hk/jormungandr/pull/3682 strictly required?

**Comment by Mr-Leshiy** [comment]
@Zeegomo , not sure, I have moved just he latest commits from master. I think it is better to ask @dkijania do we need this change there or not.

**Comment by dkijania** [comment]
@Zeegomo @Mr-Leshiy  it is not needed. just improvement for testing framework

File: jcli/Cargo.toml [hunk 1]
```diff
@@ -24,12 +24,12 @@ bech32 = "0.8"
...
-chain-addr      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-crypto    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-time    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-vote = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
+chain-core      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-impl-mockchain = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-addr      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-crypto    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-time    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-vote = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
```
Comment (Zeegomo) [inline jcli/Cargo.toml:32]: I don't think this should go in

**Review by Zeegomo — CHANGES_REQUESTED** [review]

File: jcli/Cargo.toml (hunk 1 above)
Comment (Mr-Leshiy) [reply jcli/Cargo.toml]: Of course, I have recently updated PR with the latest version from the fund7 branch :)

**Review by Zeegomo — APPROVED** [review]

**Pull request merged**